model/navigation/DefaultMessageFactory.php

```diff
@@ -25,14 +25,14 @@
 use oat\taoDelivery\model\execution\DeliveryExecutionInterface;
 use oat\taoLti\models\classes\LtiMessages\LtiMessage;
 
-class DefaultMessageFactory extends ConfigurableService
+class DefaultMessageFactory extends ConfigurableService implements LtiMessageFactoryInterface
```

Comment:
I'm not sure if it's safe to simply rename existing class which is used in configs as well. I will break our updater script.

Reply:
But it is default message factory. I don't think we should mark it as deprecated. Wee can rename it to DefaultLtiMessageFactory, but I'm not sure if it makes sense.

Reply:
it returns LtiMessage object, `DefaultMessageFactory` - name for everything :(. I was just wondering if we can rename it because you increased major version